tests/foreman/ui/test_registration.py

```diff
@@ -403,9 +409,10 @@ def test_global_registration_with_gpg_repo_and_default_package(
 
 
 @pytest.mark.tier3
+@pytest.mark.rhel_ver_match('7')
```

Comment:
Can we bump this to el8 now? or do you think it'll make sense to run this for all?

Reply:
It's already updated to el8.